components/brave_rewards/resources/ui/reducers/wallet_reducer.ts

```diff
@@ -17,6 +17,7 @@ const createWallet = (state: Rewards.State) => {
 
   chrome.send('brave_rewards.getReconcileStamp')
   chrome.send('brave_rewards.getAddresses')
+  chrome.send('brave_rewards.saveAdsSetting', ['adsEnabled', 'true'])
```

Comment:
same concern as https://github.com/brave/brave-core/pull/1282/files#r246943290. i feel it is likely we will want to allow wallet creation independent of changing adsEnabled to true.